<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportProjectDialog</class>
 <widget class="QDialog" name="ExportProjectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export Project</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="toolTip">
        <string>Defines the target architecture.

Gideros Studio supports iOS, Android, Windows Desktop, Mac OS X Desktop and WinRT (Windows Store and Windows Phone apps). Choose your 
target platform to export the application source.</string>
       </property>
       <property name="text">
        <string>Architecture:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="architecture">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>0</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>iOS</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Android</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Windows Desktop</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Mac OS X Desktop</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>WinRT</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="architectureTab">
     <property name="toolTip">
      <string>Choose your platform here.</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>iOS</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>* No extra settings for this platform.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>41</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Android</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label">
           <property name="toolTip">
            <string>Defines the package name of the Android application.

Generally speaking, the first 2 package &quot;words&quot; are your
web address in reverse.

Example: com.giderosmobile.timthetimber</string>
           </property>
           <property name="text">
            <string>Package Name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="packageName">
           <property name="text">
            <string>com.yourdomain.yourapp</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>72</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Windows Desktop</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>* No extra settings for this platform.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer1">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>41</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Mac OS X Desktop</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>* No extra settings for this platform.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>41</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
     <widget class="QWidget" name="tab1">
      <attribute name="title">
       <string>WinRT</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>
Use this to create Windows Phone and Windows Store apps ("Metro Style" apps)
A universal solution is created containing a Windows Phone and Windows Store project.

* No extra settings for WinRT.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer1">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>41</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="assetsOnly">
     <property name="toolTip">
      <string>Only exports Lua codes and assets, and not Java or Xcode source.

If you make changes to the previous exported project
on the disk, click this option to export only the Lua codes and assets.
</string>
     </property>
     <property name="text">
      <string>Export assets only</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="encryptCode">
       <property name="text">
        <string>Encrypt code files</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="encryptAssets">
       <property name="text">
        <string>Encrypt asset files</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ExportProjectDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ExportProjectDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
